dependency.lic

```diff
@@ -241,25 +241,31 @@ class ScriptManager
     @autostarts = (UserVars.autostart_scripts + Settings['autostart']).uniq
   end
 
-  def update_status_url
-    @status_url = 'https://api.github.com/repos/' + @status_repo + '/git/trees/' + @status_branch
-  end
-
-  def set_custom_status_repo(repo, branch)
-    @status_repo = repo
-    @status_branch = branch
-    update_status_url
-  end
-
-  def set_custom_status_branch(branch)
-    @status_branch = branch
-    update_status_url
+  def url_exist?(url_string)
+    url = URI.parse(url_string)
+    req = Net::HTTP.new(url.host, url.port)
+    req.use_ssl = (url.scheme == 'https')
+    path = url.path
+    res = req.request_head(path || '/')
+    res.code != "404" # false if returns 404 - not found
+  rescue Errno::ENOENT
+    false # false if can't find the server
   end
 
-  def unset_custom_status
-    @status_repo = 'rpherbig/dr-scripts'
-    @status_branch = 'master'
-    update_status_url
+  def update_status_url
+    masterurl = 'https://api.github.com/repos/' + @status_repo + '/git/trees/' + @status_branch
+    # _respond Lich::Messaging.monsterbold("Master URL is #{masterurl.inspect}")
+    mainurl   = 'https://api.github.com/repos/rpherbig/dr-scripts/git/trees/main'
```

Comment:
Shouldn't the fallback URL be `.../master`, which exists right now?

Alternatively, there's almost a fallback to the fallback, `/rpherbig/.../master`, so you could do that now to future-proof this for the duration of the handover.